### PR TITLE
Add known issues to package manual (ABC-27, ABC-91, ABC-146)

### PR DESCRIPTION
**Pull request by sebastienduverne**
1) Added known issues to package manual:
- ABC-27 - Non-convex support
- ABC-91 - Putting FBX as children of Alembic object breaks build (1192893)
- ABC-146 - Supported platform for alembic highlight

2) Pre-release review of main page (reorder/update/clean-up) - please verify Unity Editor minimum compatible version.

3) Reviewed/edited description for Package Manager, incl. mention of build target limitations

Testing: successfully built local version of the package doc using the Package Manager Docs Tool.

### CODE REVIEW

File: com.unity.formats.alembic/Documentation~/index.md [hunk 1]
```diff
@@ -17,33 +20,36 @@ The Alembic package supports these features:
...
+## Package technical details
 
-The package is available on these 64-bit desktop platforms:
-* Microsoft® Windows® 10
-* macOS® Sierra (10.12)
-* GNU/Linux (CentOS 7, Ubuntu 16.x and Ubuntu 17.x)
+### Preview package
 
-## Known limitations
+This package is in preview, so it is not ready for production use. The features and documentation in this package might change before it is verified for release.
```
Comment (clusty) [inline com.unity.formats.alembic/Documentation~/index.md:28]: The package will become re-verified briefly (alembic is already verified on a previous version). I would drop the sentence.

File: com.unity.formats.alembic/Documentation~/index.md [hunk 2]
```diff
@@ -17,33 +20,36 @@ The Alembic package supports these features:
...
+To install this package, follow the instructions in the [Package Manager documentation](https://docs.unity3d.com/Manual/upm-ui-install.html).
 
-Tell Unity about your experience using Alembic for Unity on [the Alembic-For-Unity forum](https://forum.unity.com/threads/alembic-for-unity.521649/).
+### Requirements
 
-## Installation
+This version of Alembic for Unity is compatible with the following versions of the Unity Editor:
 
-To install this package, follow the instructions in the [Package Manager documentation](https://docs.unity3d.com/Packages/com.unity.package-manager-ui@latest).
+* 2018.1 and later (recommended)
```
Comment (clusty) [inline com.unity.formats.alembic/Documentation~/index.md:38]: Suports 2019.3+

File: com.unity.formats.alembic/Documentation~/index.md [hunk 3]
```diff
@@ -17,33 +20,36 @@ The Alembic package supports these features:
...
+* Microsoft® Windows® 10
+* macOS® Sierra (10.12)
+* GNU/Linux (CentOS 7, Ubuntu 16.x and Ubuntu 17.x)
 
-1. Select a GameObject or Camera and open the Inspector view.
+### Known issues and limitations
 
-2. Click the **Add Component** button and search for **alem**.
+* Alembic for Unity **only supports** the following build targets: **Windows 64-bit**, **MacOS X** and **Linux 64-bit**.
+* Alembic does not support Material import and export.
```
Comment (clusty) [inline com.unity.formats.alembic/Documentation~/index.md:48]: Would drop/change (the format itself not supports materials.)

File: com.unity.formats.alembic/Documentation~/index.md [hunk 4]
```diff
@@ -17,33 +20,36 @@ The Alembic package supports these features:
...
+* Importing non-convex meshes results in malformed geometry (for example, triangles with flipped normals).
```
Comment (clusty) [inline com.unity.formats.alembic/Documentation~/index.md:49]: Non-convex polygons on meshes. The mesh itself can be non-convex.

**Review by clusty — CHANGES_REQUESTED** [review]

**Review by cguer — APPROVED** [review]
QA approved 👍!

**Review by clusty — APPROVED** [review]

**Pull request merged**